packages/calcite-components/src/utils/config.ts

```diff
@@ -13,8 +13,36 @@ export interface CalciteConfig {
    * @see https://github.com/focus-trap/focus-trap#createoptions
    */
   focusTrapStack: FocusTrap[];
+
+  /**
+   * Contains the version of the Calcite components.
+   */
```

Comment:
maybe we can document this as readOnly somehow?

Reply:
We can add `@readonly` to the interface doc, but it won't show up in a readme nor the `docs-json.json` file. @geospatialem @DitwanP @macandcheese Do you have any suggestions?